net/http/authz/non_loopback_authz.go

```diff
@@ -0,0 +1,9 @@
+//+build !loopback_auth
+
+package authz
+
+func init() {
+	builtinGrants = append(builtinGrants,
+		&Grant{GuardType: "any", Policy: "public"},
```

Comment:
I suggest moving this to the definition of builtin grants in authorizer.go (and so take it out of loopback_authz.go too).